core/src/web/camera.ts

```diff
@@ -16,29 +18,104 @@ export class CameraPluginWeb extends WebPlugin implements CameraPlugin {
   }
 
   async getPhoto(options: CameraOptions): Promise<CameraPhoto> {
-    options;
-
     return new Promise<CameraPhoto>(async (resolve, reject) => {
-      const cameraModal: any = document.createElement('pwa-camera-modal');
-      document.body.appendChild(cameraModal);
-      await cameraModal.componentOnReady();
-      cameraModal.addEventListener('onPhoto', async (e: any) => {
-        const photo = e.detail;
-
-        if (photo === null) {
-          reject('User cancelled photos app');
-        } else if (photo instanceof Error) {
-          reject(photo.message);
+      if (options.webUseInput === false && options.source === CameraSource.Camera) {
+        const cameraModal: any = document.createElement('pwa-camera-modal');
```

Comment:
`cameraModal` can be undefined if PWA Elements isn't installed, do we want to add a message about it?